### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by MDVAurelien**
Add some French translations

### CODE REVIEW

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -117,55 +117,55 @@
...
+  "Docker Images": "Images Docker",
   "Does your business need <a href=\"https://www.ixsystems.com/truenas/?utm_source=truenas+core+ui&utm_medium=os&utm_campaign=welcome\" target=\"_blank\">Enterprise level</a> support and services? Contact <a href=\"https://www.ixsystems.com/quote-form/?utm_source=truenas+core+ui&utm_medium=os&utm_campaign=welcome\"target=\"_blank\">iXsystems</a> for more information.": "",
-  "Download Updates": "",
-  "Download actions": "",
-  "Drive Details": "",
-  "Drive Temperatures": "",
+  "Download Updates": "Télécharger les mises à jour",
+  "Download actions": "Actions de téléchargement",
+  "Drive Details": "Détails du lecteur",
+  "Drive Temperatures": "Températures du lecteur",
```
Comment (RehanY147) [inline src/assets/i18n/fr.json:168]: Google translate shows this doesn't exactly translate to what the text says. Please reconsider. Same goes for other similar strings.

**Review by RehanY147 — CHANGES_REQUESTED** [review]

File: src/assets/i18n/fr.json (hunk 1 above)
Comment (MDVAurelien) [reply src/assets/i18n/fr.json]: OK, done

Comment (MDVAurelien) [reply src/assets/i18n/fr.json]: Hello, is it possible to validate in order to continue the translation?

**Comment by undsoft** [comment]
@RehanY147 Please resolve conflicts and merge.

**Pull request closed without merging**